cpp/src/io/parquet/page_hdr.cu

```diff
@@ -356,16 +340,19 @@ struct gpuParsePageHeader {
  */
 // blockDim {128,1,1}
 __global__ void __launch_bounds__(128)
-  gpuDecodePageHeaders(ColumnChunkDesc* chunks, int32_t num_chunks)
+  gpuDecodePageHeaders(ColumnChunkDesc* chunks, int32_t num_chunks, int32_t* error_code)
 {
   gpuParsePageHeader parse_page_header;
   __shared__ byte_stream_s bs_g[4];
 
+  int error[4]            = {0};
```

Comment:
Do we need to be explicit about `int` vs. `int32_t`, for the purpose of aligning our conventions? I'd like to use the same type for the `error` value as the type we use in the parameters of `set_error`.